docs/pages/Architecture/offloading/features/computing.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Computing
+weight: 3
+---
+
+### Overview
+
+The offloading of a pod in a foreign cluster involves a two-steps scheduling process, which can be summarized as follows:
+- the scheduler of the home cluster selects the _big node_ as the best target to run the pod;
+- the big node, being virtual, cannot run any pod. Hence, the virtual kubelet handling the big node involves the scheduler of the foreign cluster, which will select the best (physical) node on the foreign cluster to run the pod.
+
+In addition, the proper reconciliation processes are executed by the Liqo virtual kubelet in order to allow the home cluster to be up-to-date with the local representation of the remote pod, to ensure complete visibility on the Pod albeit running in the remote cluster.
+
+
+### Resiliency of the remote pod 
+
+Due to the properties of the two-step scheduling process, different behaviours can be spotted when deleting a pod running in a foreign cluster:
+- The pod is deleted in the **home** cluster (intentionally or by eviction): the remote one has to be deleted immediately. In fact, the home cluster is the owner of Pod, therefore whatever modification to the pod is set, this has to be reflected on the remote pod. Hence, when the pod is deleted in the home cluster, the status of the remote cluster status has to be aligned immediately, leading to the deletion of the remote Pod.
+- The pod is deleted in the **foreign** cluster (intentionally or by eviction): not only the local pod must not be deleted, but the remote pod has to be re-created as soon as possible. In fact, the two-steps scheduling process does not allow the foreign cluster to trigger a re-scheduling in the home cluster; the requested pod must be running in the foreign cluster, without involving the home cluster in any un-necessary detail, unless an unrecoverable condition is detected.
+
+To implement this behavior, we have to remember that, in Kubernetes, pods are commonly created starting from a [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) or a [Job](https://kubernetes.io/docs/concepts/workloads/controllers/job/).
+However, in Liqo, the request for a new deployment scheduled on a foreign cluster triggers the Liqo controller-manager (running in the virtual kubelet) to create a new [ReplicaSet](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/), that ultimately leads to the creation of the Pod (again, through the controller-manager).
```

Comment:
```suggestion
However, in Liqo, the request for a new deployment scheduled on a foreign cluster triggers the Liqo controller-manager (running in the virtual kubelet) to create a new [ReplicaSet](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/), that ultimately leads to the creation of the Pod (again, through the controller-manager).
```

I suppose this should be rephrased as AFAIU Liqo does not deal with Deployments and ReplicaSets (managed by the vanilla controller-manager), but with the pods created by those.